src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -69,7 +76,7 @@ function KeylessBackupProgress({
   if (route.params.keylessBackupFlow === KeylessBackupFlow.Restore) {
     return <Restore />
   } else {
-    return <Setup />
+    return <Setup navigatedFromSettings={navigatedFromSettings} />
```

Comment:
I wonder if something such as `isOnboarding` would be clearer; however, that also gets a bit muddled when someone could technically be onboarding to CAB from settings.

Reply:
I changed what was being passed so now origin is being passed (to make analytics easier). Now navigatedFromSettings is computed in the Setup component, can change the name if you want but I'm not sure if changing would make any more clear?